gensim/matutils.py

```diff
@@ -505,21 +505,22 @@ def arun_metric(min_num_topics=10, max_num_topics=50, iterations=10):
     Thanks to Adrien Guille for the implementation of the metric.
     
     """
-    kl_matrix = []
-    for j in xrange(iterations):
-        kl_list = []
-        l = np.array([sum(corpus.vector_for_document(doc_id)) for doc_id in xrange(corpus.size)])  # document length
-        norm = np.linalg.norm(l)
-        for i in xrange(min_num_topics, max_num_topics + 1):
-            c_m1 = np.linalg.svd(self.topic_word_matrix.todense(), compute_uv=False)
-            c_m2 = l.dot(self.document_topic_matrix.todense())
-            c_m2 += 0.0001  # we need this to prevent components equal to zero
-            c_m2 /= norm
-            kl_list.append(symmetric_kl(c_m1.tolist(), c_m2.tolist()[0]))
-        kl_matrix.append(kl_list)
-    output = np.array(kl_matrix)
-    return output.mean(axis=0)
-
+    results = []
+    for i in range(min_topics, max_topics, iteration):
```

Comment:
For big models, it needs a lot of time. It doesn't look like very useful feature for real tasks.

Reply:
Agree on this. I thought of taking the trained model as a parameter, but that makes it difficult to evaluate the range